variables.tf

```diff
@@ -127,6 +127,12 @@ variable "licenseServerMicrosoftMonitoringAgent" {
   default     = true
 }
 
+variable "licenseServerMicrosoftGuestConfiguration" {
+  type        = bool
+  description = "Specifies whether a Microsoft Guest configuration extension will be installed on license server VM."
```

Comment:
This variable depends on the licenseServer variable. I'd find it nice if this would be mentioned in the description - btw the same applies for the variables licenseServerIaaSAntimalware and licenseServerMicrosoftMonitoringAgent.